doc/DoxygenLayout.xml

```diff
@@ -5,6 +5,7 @@
     <tab type="mainpage" visible="yes" title=""/>
     <tab type="user" visible="yes" title="Tutorial" url="https://github.com/ginkgo-project/ginkgo/wiki/Tutorial:-Building-a-Poisson-Solver" />
     <tab type="user" visible="yes" title="Examples" url="@ref Examples" />
+    <tab type="user" visible="yes" title="Citing" url="@ref citing_ginkgo" />
     <tab type="user" visible="yes" title="Contributing" url="@ref contributing_guidelines" />
```

Comment:
Maybe we can change both of these to maintain uniformity with the Using Ginkgo tab ?   
```suggestion
    <tab type="user" visible="yes" title="Citing Ginkgo" url="@ref citing_ginkgo" />
    <tab type="user" visible="yes" title="Contributing to Ginkgo" url="@ref contributing_guidelines" />
```